<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>571</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>909</width>
       <height>569</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>481</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="add_dic"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_2"/>
   <addaction name="add_dic"/>
   <addaction name="del_node"/>
   <addaction name="action_4"/>
   <addaction name="separator"/>
   <addaction name="enlarge"/>
   <addaction name="smalal"/>
   <addaction name="resize"/>
   <addaction name="action_8"/>
   <addaction name="action_11"/>
   <addaction name="action_9"/>
   <addaction name="action_10"/>
   <addaction name="exit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>图片资源管理器</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>245</width>
       <height>471</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <column>
      <property name="text">
       <string>节点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>节点类型</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>图片文档</string>
      </property>
      <item>
       <property name="text">
        <string>分组节点</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>分组2</string>
       </property>
      </item>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="add_dic">
   <property name="text">
    <string>添加目录</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>添加文件</string>
   </property>
  </action>
  <action name="del_node">
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>show_node</string>
   </property>
  </action>
  <action name="enlarge">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="smalal">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="resize">
   <property name="text">
    <string>实际大小</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>适合宽度</string>
   </property>
   <property name="toolTip">
    <string>adapt_wide</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>适合高度</string>
   </property>
   <property name="toolTip">
    <string>adapt_height</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>窗体浮动</string>
   </property>
   <property name="toolTip">
    <string>window_float</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>窗体可见</string>
   </property>
   <property name="toolTip">
    <string>window_stowable</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
